app/helpers/react_on_rails_helper.rb

```diff
@@ -103,13 +104,13 @@ def react_component(component_name, raw_options = {})
 
     # Setup the page_loaded_js, which is the same regardless of prerendering or not!
     # The reason is that React is smart about not doing extra work if the server rendering did its job.
-
+    
     component_specification_tag =
-      content_tag(:div,
-                  "",
-                  class: "js-react-on-rails-component",
-                  style: options.style,
-                  data: options.data)
+    content_tag(:div,
```

Comment:
This was indented before to indicate that the content_tag is assigned to the preceding line.